src/arviz_stats/base/dataarray.py

```diff
@@ -240,5 +240,16 @@ def thin(self, da, factor="auto", dims=None):
 
         return da.sel({dims: slice(None, None, factor)})
 
+    def pareto_min_ss(self, da, dims=None):
```

Comment:
The array function already supports 1d input (aka only `sample` dimension instead of `chain, draw`). But this one won't. I tried adding support for that in `ess` at https://github.com/arviz-devs/arviz-stats/pull/22/files to unblock `plot_ess`, do you think it is ok to use that convention with `dims` being either length 1 or 2?

If so I could also make a helper/validate function to take care of all that in the other PR and merge it, then it can be used here.

Reply:
ok